doc/docbook/nmsgtool.1

```diff
@@ -430,6 +443,20 @@ are required\&.
 Write NMSG payloads to a file\&.
 .RE
 .PP
+\fB\-W\fR \fIkafka\fR, \fB\-\-writekafka\fR \fIkafka\fR
+.RS 4
+Write NMSG payloads to Kafka\&. The address
+\fIkafka\fR
+has format Topic#Partition@Broker,Offsetis.
+.sp
+.sp
+Examples of possible
+\fIkafka\fR
+arguments to
+\fB\-K\fR
+include "ch202#0@kafka.local.com:9092" to indicate that nmsgtool shall write to topic ch202 on partition 0 at offset 3000 to Kafka broker kafka.local.com..
```

Comment:
- [ ] - Remove extra trailing period.
- [ ] - Does nmsgtool use this offset?

Reply:
When clicking through to your comment it looks like you're working with an old commit. I am guessing this is also the reason it says above, "doc/docbook/nmsgtool.1 (Outdated)"